client/my-sites/plugins-wpcom/jetpack-plugins.js

```diff
@@ -0,0 +1,187 @@
+/**
+ * External dependencies
+ */
+import { identity } from 'lodash';
+
+const jetpackPlugins = ( translate = identity ) => [
+	{
+		category: 'engagement',
+		name: translate( 'Engagement' ),
+		icon: 'stats-alt',
+		plugins: [
+			{
+				name: translate( 'Stats' ),
+				link: 'https://support.wordpress.com/stats/',
+				description: translate( 'View your site\'s visits, referrers, and more.' ),
+				plan: 'standard',
```

Comment:
Should we rename this from `standard` to `free`, in order to match it with the actual plan name?